packages/boot/test/bootstrapTests/test-liquidation-concurrent-2b.ts

```diff
@@ -0,0 +1,486 @@
+  await advanceTimeBy(3, 'minutes');
+
+  console.log('step 3 of 10');
+  await advanceTimeBy(3, 'minutes');
+
+  console.log('step 4 of 10');
+  await advanceTimeBy(3, 'minutes');
+
+  console.log('step 5 of 10');
+  await advanceTimeBy(3, 'minutes');
+
+  cases.map(async ({ collateralBrandKey, managerIndex }) => {
+    t.like(readLatest(`published.auction.book${managerIndex}`), {
+      collateralAvailable: {
+        value: scale6(setups[collateralBrandKey].auction.start.collateral),
+      },
+    });
+  });
+
+  console.log('step 6 of 10');
+  await advanceTimeBy(3, 'minutes');
+
+  console.log('step 7 of 10');
+  await advanceTimeBy(3, 'minutes');
+
+  console.log('step 8 of 10');
+  await advanceTimeBy(3, 'minutes');
+
+  console.log('step 9 of 10');
+  await advanceTimeBy(3, 'minutes');
+
+  console.log('step 10 of 10');
+  await advanceTimeBy(3, 'minutes');
+
+  console.log('step 11 of 10');
+  await advanceTimeBy(3, 'minutes');
+
+  cases.map(async ({ collateralBrandKey, managerIndex }) => {
+    check.vaultNotification(managerIndex, 0, {
+      debt: undefined,
+      vaultState: 'active',
+      locked: {
+        value: scale6(outcomes[collateralBrandKey].vaultsActual[0].locked),
+      },
+    });
+    check.vaultNotification(managerIndex, 1, {
+      debt: undefined,
+      vaultState: 'active',
+      locked: {
+        value: scale6(outcomes[collateralBrandKey].vaultsActual[1].locked),
+      },
+    });
+    check.vaultNotification(managerIndex, 2, {
+      debt: undefined,
+      vaultState: 'liquidated',
+      locked: {
+        value: scale6(outcomes[collateralBrandKey].vaultsActual[2].locked),
+      },
+    });
+  });
+
+  const metricsPathA = metricsPaths[managerIndexA];
+  const metricsPathSt = metricsPaths[managerIndexSt];
+
+  // ATOM
+  t.like(readLatest(metricsPathA), {
+    // reconstituted
+    numActiveVaults: 2,
+    numLiquidationsCompleted: 1,
+    numLiquidatingVaults: 0,
+    retainedCollateral: { value: 0n },
+    totalCollateral: { value: 29795782n },
+    totalCollateralSold: { value: 13585013n },
+    totalDebt: { value: 204015000n },
+    totalOverageReceived: { value: 0n },
+    totalProceedsReceived: { value: 100000000n },
+    totalShortfallReceived: {
+      value: scale6(outcomes[collateralBrandKeyA].reserve.shortfall),
+    },
+  });
+
+  // STARS
+  t.like(readLatest(metricsPathSt), {
+    // reconstituted
+    numActiveVaults: 2,
+    numLiquidationsCompleted: 1,
+    numLiquidatingVaults: 0,
+    retainedCollateral: { value: 0n },
+    totalCollateral: { value: 29796989n },
+    totalCollateralSold: { value: 12348888n },
+    totalDebt: { value: 223110000n },
+    totalOverageReceived: { value: 0n },
+    totalProceedsReceived: { value: 100000000n },
+    totalShortfallReceived: {
+      value: scale6(outcomes[collateralBrandKeySt].reserve.shortfall),
+    },
+  });
+};
+
+// Reference: Flow 2b from https://github.com/Agoric/agoric-sdk/issues/7123
+test.serial('scenario: Flow 2b', async t => {
```

Comment:
the helper function factoring is for re-use in multiple tests. since there's only in in this case it's an unwarranted complication